src/open_inwoner/components/templates/components/Header/NavigationAuthenticated.html

```diff
@@ -34,9 +41,8 @@
 {% elif config.login_show %}
     <div class="desktop-login">
         <span class="desktop-login__link">
-            {% url 'login' as login_url %}
             {% trans "Inloggen" as login %}
-            {% button text="Inloggen" href=login_url icon="person" icon_position="before" primary=True icon_outlined=True %}
+            {% button text="Inloggen" href="login" icon="person" icon_position="before" primary=True icon_outlined=True %}
```

Comment:
Same again... trans tag is not used
```suggestion
            {% button text=_("Inloggen") href="login" icon="person" icon_position="before" primary=True icon_outlined=True %}
```